### PR TITLE
Technigo Boot Camp project week 5: Guess Who?

### PR DESCRIPTION
**Pull request by idasylven**
The project:

After some major struggles I ultimately managed to fulfill the requirements for the Blue Level. I must admit this week has been incredibly hard for me in terms of the JavaScript understandings. My current metaphor is that I´m on a JavaScript highway and all my brain is providing me with is a tricycle. I have a hard time keeping up in general, and do not believe I would have been able to succeed this weeks requirements without the help and support from my team members - I´m forever grateful! And I really hope I will be able to return the favor in the upcoming weeks. I´m re-watching old tutorials, doing online JavaScript exercises and trying all that I can to improve my understandings. Fingers crossed!

Red Level:

I added some conceptual 80s styling to my project using CSS. If I had more time I would have liked to change all board characters to eclectic artists from that era (see details in README section).

Black Level:

I have added sound effects (also 80s themed) to two event occasions in the game:

1. After the character has selected a character feature, gotten the alert and the game board is re-drawn there are two different sound effects. They are based on if the feature was correct or not in comparison to the secret person.
2. When the Win or Loose section of the game is displayed there are two different sound effects based on if the player did indeed win or not.

Link to live web page:
https://reverent-hawking-ae4148.netlify.app/

### CODE REVIEW

**Review by sandra-wallen — COMMENTED** [review]
**Approved**

I love the 80's design and the sound effects, really nice that you added your own styling to the game! 

I can tell both from the design, the alerts and other texts seen on your website that you think through your ideas and choice of words, and also in the README where you have a clear picture of what you would implement next. That's really good!

I see that you have added descriptional comments in the HTML and adjusted the ones in the JS file, this is very helpful and I could follow and understand your code easily.

Great job, your game looks really cool! :raised_hands: :star2:

File: code/script.js [hunk 1]
```diff
@@ -2,236 +2,243 @@
...
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOutbtn = document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose')
+const winOrLoseText = document.getElementById('winOrLoseText')
+const playAgainbtn = document.getElementById('playAgain')
+const losingSound = document.getElementById('losingSound')
+const winningSound = document.getElementById('winningSound')
+const declinedSound = document.getElementById('declinedSound')
+const confirmationSound = document.getElementById('confirmationSound')
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:12]: Those are not normal variables but pointers to the DOM elements. In vanilla js, they need to be handled this way. 

Other variables should preferably be not global ones. And if you can, always avoid creating variables without assigned starting value.

File: code/index.html [hunk 2]
```diff
@@ -1,64 +1,85 @@
...
-          class="guess-who-icon"
-          src="images/guess-who-bubble.png"
-          alt="Guess Who"
-        />
-        <h1 id="winOrLoseText"></h1>
-        <button id="playAgain" class="filled-button">PLAY AGAIN</button>
-      </div>
+        <div class="win-or-lose">
+            <h1 id="winOrLoseText"></h1>
+            <button id="playAgain" class="filled-button">PLAY AGAIN</button>
```
Comment (KarolCislowski) [inline code/index.html:72]: I was clicking this button but nothing happened 😭

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

Remember that you can always come back to it when you have some spare time and add more black level functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**